rules/CodeQuality/Rector/Class_/ControllerMethodInjectionToConstructorRector.php

```diff
@@ -31,6 +31,11 @@
  */
 final class ControllerMethodInjectionToConstructorRector extends AbstractRector
 {
+    /**
+     * @var string[]
+     */
+    private const COMMON_ENTITY_CONTAINS_SUBNAMESPACES = ['\\Entity', '\\Document', '\\Model'];
```

Comment:
The `\\` prefix somehow removed under single quote on scoped build

https://github.com/rectorphp/rector/commit/100442a966e4ef2012191bc6c45fc2f7384d2a4d#diff-2beb8610233424551baf38f6ba20dec69759c01de3abaa2ef4bd0f7f668079f1

On our usecase, this is still ok, but for future string related backslash usage that important(eg: regex), the double quote should be used.